src/main/java/bio/terra/service/resourcemanagement/BufferService.java

```diff
@@ -105,7 +79,7 @@ public ResourceInfo handoutResource(boolean enableSecureMonitoring) {
     logger.info("Using request ID: {} to get project from RBS", handoutRequestId);
     HandoutRequestBody requestBody = new HandoutRequestBody().handoutRequestId(handoutRequestId);
     try {
-      BufferApi bufferApi = bufferApi(bufferServiceConfiguration.getInstanceUrl());
```

Comment:
This seemed redundant -- we can access the config when we create the clients and only set the base path in one place.